resources/core/charts/configurations-generator/README.md

```diff
@@ -15,4 +15,31 @@
 
 ## Overview
 
-This is a generator of configurations used in Kyma. 
+This chart installs the Kubeconfig generator, a proprietary tool that generates a `Kubeconfig` file which allows the user to access to the Kyma cluster through the Command Line Interface (CLI), and to manipulate the connected cluster within the permission boundaries of the user.
```

Comment:
1. to access ~to~ the Kyma cluster
2. 2. Is "manipulate" a good word here? Maybe "manage" would fit better?